d3ledger/back-office
# D3-811 | Update login and signup pages

# Description
https://soramitsu.atlassian.net/browse/D3-811

#### Works done
- [x] Updated login page
- [x] Updated signup page
- [x] Removed css styles to global
- [x] Applied one font for all elements

# Screenshots
![image](https://user-images.githubusercontent.com/16295803/51983886-ef56e380-24aa-11e9-8fb2-0c2bde2fed4e.png)
![image](https://user-images.githubusercontent.com/16295803/51983927-0eee0c00-24ab-11e9-904e-c4125fe54d88.png)
![image](https://user-images.githubusercontent.com/16295803/51983915-039ae080-24ab-11e9-884c-0d1e4af7a3b5.png)

# How to test
1. `yarn` to install dependencies
2. `yarn serve` to run application
3. Check login and signup pages
